backend/test/fixtures/pbmc3k-genesets.csv

```diff
@@ -1,16 +1,21 @@
 # Test fixture
 gene_set_name,gene_set_description,gene_symbol,gene_description
-first gene set name,,F5, a gene_description
+first gene set name,a description,F5, a gene_description
```

Comment:
why did you change the descriptions on line 3, 6, and 7?  This changes the test (ie, it no longer validates that we correctly merge different descriptions for a gene set). If you have tests that require this change, please add a new gene set to test it on.